Makefile

```diff
@@ -207,7 +208,16 @@ setup-new-sdk:
 .PHONY: build-client-java
 build-client-java:
 	make build-client sdk_language=java tmpdir=${TMP_DIR}
-	make run-in-docker sdk_language=java image=gradle:${GRADLE_DOCKER_TAG} command="/bin/sh -c 'gradle fmt'"
+	make run-in-docker sdk_language=java image=busybox:${BUSYBOX_DOCKER_TAG} command="/bin/sh -c 'chmod +x ./gradlew'"
+	make run-in-docker sdk_language=java image=gradle:${GRADLE_DOCKER_TAG} command="/bin/sh -c './gradlew fmt'"
```

Comment:
Would it make sense to run build gradle task in the same command?  This may improve speed by not requiring spinning up the same docker twice

Reply:
Oh, yes good call

```suggestion
	make run-in-docker sdk_language=java image=gradle:${GRADLE_DOCKER_TAG} command="/bin/sh -c 'chmod +x ./gradlew && ./gradlew fmt build'"
```

Reply:
Along these lines, calling `./gradlew` at all forces a download, so I should avoid calling the [Gradle wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html) altogether and use `gradle` from the Docker image.

Reply:
Consolidated these in [2080d74](https://github.com/openfga/sdk-generator/pull/160/commits/2080d74e72d5b19f242c1bcaee3585a9fc93fd00)